org-policy/.manifest

```diff
@@ -0,0 +1,3 @@
+{
+    "roots": ["diamond"]
```

Comment:
Will this collide with the `diamond.data` of `bundler`?

Reply:
Moved to `diamond.policy` to avoid colliding